packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -166,6 +169,28 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ```
 /// {@end-tool}
 ///
+/// {@tool sample}
+///
+/// This example of a red [Container] wrapping a [GestureDetector] widget. On
+/// this example the since the [GestureDetector] does not have a child it
+/// assumes the size of its parent, making the area of the surrounding
+/// [Container] clickable.
```

Comment:
Same note about the line break here.

Reply:
Do you refer to the line break at the line 177 or all of them (176, 174) as well?
I added just so it wouldn't pass 80 columns as the others docs.

Reply:
I only mean the line breaks on liens that are not close to the 80 char limit. in this example here that's line 177.